clean.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import logging
+from datasets import Dataset, load_dataset
+
+from datasets.utils.logging import set_verbosity_info
+
+set_verbosity_info()
+logger = logging.getLogger(__name__)
+
+
+# Map functions
+MAPS = {}
+# Filter functions
+FILTERS = {}
+
+assert set(MAPS.keys()).isdisjoint(set(FILTERS.keys()))
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--dataset-path",
+        type=str,
+        required=True
+    )
+    parser.add_argument(
+        "--maps-and-filters",
+        nargs="+",
+        type=str,
+        required=True
+    )
+    parser.add_argument(
+        "--save-dataset",
+        type=str,
+        required=True
+    )
+    parser.add_argument(
+        "--num-proc",
+        type=int,
+        default=1
+    )
+    args = parser.parse_args()
+    return args
+
+# TODO: think of a batch mechanism
```

Comment:
if you want to apply the functions in batch mode we could just set 
```Python
ds.map(map_function, num_proc=num_proc, batched=True, batch_size=batch_size)
```

Reply:
we should probably either do all batched or none, since this effects how the function is called and we need to change the logic. 

alternatively we could define each map/filter as a class with a `__call__` method that could have some attributes such as `self.batched=True`.

Reply:
i'd be in favor of everyone in batch, I can't seem to think of any functions we'd want not to apply in batch / multiprocessing